pubspec.yaml

```diff
@@ -15,6 +15,8 @@ dependencies:
   cupertino_icons: ^1.0.2
   presentation:
     path: packages/presentation
+  domain:
+    path: packages/domain
```

Comment:
presentation already contains domain

Reply:
yes, I did it because in injector.dart file linter underscored my import =)

Reply:
try to execute pub get in all modules